packages/styleguide/src/templates/Front/index.js

```diff
@@ -433,21 +432,19 @@ const createFrontSchema = ({
   const frontTileTeaser = {
     matchMdast: matchTeaserType('frontTile'),
     component: ({ children, attributes, ...props }) => (
-      <ColorContextProvider colorSchemeKey='light'>
```

Comment:
Where is the '.light' class?

Reply:
It's a) not needed because tile colors are defined via CMS and b) the additional div would mess up the tile layout.